<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyConstraintActivitiesPreferredStartingTimesForm_template</class>
 <widget class="QDialog" name="ModifyConstraintActivitiesPreferredStartingTimesForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="This is the title of the dialog to modify a constraint of this type">Modify activities preferred starting times</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="filterGridLayout">
     <item row="1" column="0">
      <widget class="QComboBox" name="teachersComboBox"/>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="activityTagsComboBox"/>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="subjectsComboBox"/>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="activityTagsTextLabel">
       <property name="text">
        <string>Activity tag</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="teachersTextLabel">
       <property name="text">
        <string>Teacher</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="studentsComboBox"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="studentsTextLabel">
       <property name="text">
        <string>Students set</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="subjectsTextLabel">
       <property name="text">
        <string>Subject</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="durationCheckBox">
       <property name="text">
        <string>Duration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="durationSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="preferredTimesTable"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="groupTextLabel">
       <property name="text">
        <string>Group:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="essentialRadioButton">
       <property name="text">
        <string>Essential</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="importantRadioButton">
       <property name="text">
        <string>Important</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="desirableRadioButton">
       <property name="text">
        <string>Desirable</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="weightBoxLayout">
     <item>
      <widget class="QLabel" name="weightTextLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Weight percentage (recommended: 0.0%-100.0%)</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="weightLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string notr="true">100</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="buttonsGridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="setAllAllowedPushButton">
       <property name="text">
        <string comment="It refers to time slots">Set all allowed</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string extracomment="This is an explanation in a dialog for a constraint. It says that symbol X means that this slot is not allowed, and an empty cell means that the slot is allowed">X=not allowed, empty=allowed</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="setAllNotAllowedPushButton">
       <property name="text">
        <string comment="It refers to time slots">Set all not allowed</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
